state_handlers.js

```diff
@@ -193,6 +382,40 @@ function getTitlesHelper(stateObj) {
   );
 }
 
+function getTitleChunk(articleJson, stateObj) {
+  let articleLen = Object.keys(articleJson).length;
+  console.log('Length of array is: ' + articleLen);
+  let arrArticles = Object.keys(articleJson);
+  let retSpeech = '<break />';
+  let j = 0;
+  console.log(
+    'stateObj.attributes[titleCount] is: ' + stateObj.attributes['titleCount']
+  );
+  for (
+    let i = stateObj.attributes['titleCount'];
+    i < stateObj.attributes['titleCount'] + constants.TITLE_CHUNK_LEN &&
+    i < articleLen;
+    i++
+  ) {
+    console.log('article title: ' + articleJson[i].title);
+    let num = i + 1;
+    retSpeech +=
+      num +
+      '.  ' +
+      articleJson[i].title +
+      '. ' +
+      articleJson[i].lengthMinutes +
+      ' minutes.  ';
```

Comment:
Just a style thing but I think templates make this kind of string construction easier to read:
```
retSpeech = `${retSpeech} ${i+1}. ${articleJson[i].title}. ${articleJson[i].lengthMinutes} minutes.  `
```

Reply:
Thanks.  Refactored:
``
function getTitleChunk(articleJson, stateObj) {
  let retSpeech = '<break />';

  let curCount = stateObj.attributes['titleCount'];
  let arrChunk = Object.keys(articleJson).slice(
    curCount,
    curCount + constants.TITLE_CHUNK_LEN
  );

  arrChunk.forEach(function(element, index) {
    console.log('article title: ' + arrChunk[index].title);

    retSpeech = `${retSpeech} ${index + 1}. ${arrChunk[index].title}. ${
      arrChunk[index].lengthMinutes
    } minutes.  `;
  });
  stateObj.attributes['titleCount'] += arrChunk.length;

  return retSpeech;
}

``

Reply:
Cool, nice improvement. And in the foreach, can you use `element` instead of `arrChunk[index].`?

Reply:
Yes, I realized that after I posted this, so that is in the PR that I pushed.